folium/folium.py

```diff
@@ -676,11 +663,9 @@ def _popup_render(self, popup=None, mk_name=None, count=None,
             Type of marker. Simple, Circle, etc.
         count: int, default None
             Count of marker
-        popup_on: boolean, default True
-            If False, no popup will be rendered
         """
-        if not popup_on:
-            return 'var no_pop = null;'
+        if not popup:
+            return ''
```

Comment:
@themiurgo Quick questions before we merge this.  Why you choose an empty string instead of the `'var no_pop = null;'`  Will the marker be clickable with the empty string?  If so, that might be misleading.